### PR TITLE
SEO-172168-AngularJS-Control Missing H1

### PR DESCRIPTION
**Pull request by Omanyadickson**
Hi. Girija-Rajendran,

Kindly review.
Modifying missing H1. in my PR below.

Regards.
Dickson J

### CODE REVIEW

File: angularjs/ListView/Getting-Started.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting started | ListView | JavaScript | Syncfusion
-description: Getting started with listview widget in angularjs
+title: Getting Started with AngularJS ListView control | Syncfusion
```
Comment (Sharly-Atieno) [inline angularjs/ListView/Getting-Started.md:3]: Control (c should be in caps)

File: angularjs/ListView/Getting-Started.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+description:Learn here all about Getting started with Syncfusion Essential AngularJS ListView control, its elements, and more.
```
Comment (Sharly-Atieno) [inline angularjs/ListView/Getting-Started.md:4]: Update as below
description: Learn here all about Getting started with Syncfusion Essential AngularJS ListView control, its elements, and more.
image

File: angularjs/ListView/Getting-Started.md (hunk 1 above)
Comment (Omanyadickson) [reply angularjs/ListView/Getting-Started.md]: renamed

**Review by MallikaRK — APPROVED** [review]
Approved

File: angularjs/ListView/Getting-Started.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Getting started | ListView | JavaScript | Syncfusion
-description: Getting started with listview widget in angularjs
+title: Getting Started with AngularJS ListView Control | Syncfusion
+description:Learn here all about getting started with Syncfusion Essential AngularJS ListView control, its elements, and more.
```
Comment (Girija-Rajendran) [inline angularjs/ListView/Getting-Started.md:4]: space issue
image

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**